sql/4_2_2-to-4_3_1_upgrade.sql

```diff
@@ -743,3 +743,10 @@ ALTER TABLE `users` DROP COLUMN `ssi_relayhealth`;
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `notes`, `activity`) VALUES ('page_validation', 'common#new_encounter', '/interface/forms/newpatient/common.php', 50, '{pc_catid:{exclusion: ["_blank"]}}', 1);
 #EndIf
 
+#IfMissingColumn insurance_companies ins_type_code
```

Comment:
In terms of avoiding a mysql crash, probably better to use(if the user customized the table and removed freeb_type):
``````
#IfColumn insurance_companies freeb_type